### PR TITLE
docs: update monitoring dashboard links

### PR DESCRIPTION
**Pull request by andrewbrook**
Update text to reference name of monitoring product and link to dashboard

Checklist (if applicable):
- [X] PR title is following https://www.conventionalcommits.org/en/v1.0.0/
- [X] Docs updated (updated docs or a docs bug required)

### CODE REVIEW

File: docs/monitoring.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Firebase Genkit is fully instrumented with
 
 ## Telemetry Configuration
 
-Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin poweres the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) that has an AI-idiomatic view of telemetry data.
+Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin powers the [Genkit Monitoring dashboard (public preview)](https://firebase.google.com/u/0/project/_/genai_monitoring) that has an AI-idiomatic view of telemetry data.
```
Comment (pavelgj) [inline docs/monitoring.md:8]: `https://firebase.google.com/u/0/project/_/genai_monitoring` is not a working link

Comment (MichaelDoyle) [reply docs/monitoring.md:8]: As an aside - should we just drop the "public preview" label? or is it important to have that in the docs? Seems like something that will go stale.

Comment (andrewbrook) [reply docs/monitoring.md]: Whoops, fixed the link. Removed public preview too. I think that makes sense given the the Console will still show this message while it is relevant.

**Review by tonybaroneee — APPROVED** [review]

File: docs/monitoring.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ Firebase Genkit is fully instrumented with
 
 ## Telemetry Configuration
 
-Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin poweres the [Firebase AI Monitoring dashboard (private preview)](https://forms.gle/Lp5S1NxbZUXsWc457) that has an AI-idiomatic view of telemetry data.
+Genkit automatically manages tracing and metrics without requiring explicit configuration. You can enable telemetry exports for Firebase or Google Cloud using their respective plugins and helper functions. Using either plugin powers the [Genkit Monitoring dashboard](https://console.firebase.google.com/project/_/genai_monitoring) that has an AI-idiomatic view of telemetry data.
```
Comment (tonybaroneee) [inline docs/monitoring.md:8]: `AI-centric` instead?

**Review by tonybaroneee — APPROVED** [review]

**Pull request merged**